pipenv/core.py

```diff
@@ -2791,8 +2792,9 @@ def do_check(
 
 
 def do_graph(project, bare=False, json=False, json_tree=False, reverse=False):
+    import json as jsonlib
```

Comment:
Can we just `import json` at the top level now?  I don't really see why to namespace it to `jsonlib` either.